source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -383,6 +389,60 @@ namespace
         printAssistedCapture3DResults(allDurations);
     }
 
+    
+    template<typename DataType>
+    void CopyDataTime(Zivid::PointCloud pointCloud, DataType copyname)
+    {
+        std::vector<Duration> copyDataDurations;
+
+        const auto beforecopyData = HighResClock::now();
+        const auto tempcopy = pointCloud.copyData<decltype(copyname)>();
+        const auto aftercopyData = HighResClock::now();
+
+        copyDataDurations.push_back(aftercopyData - beforecopyData);
+        
+        std::cout << copyname << std::endl;
```

Comment:
Maybe you don't need this line. You print header outside this function (before calling it).

Reply:
I will delete line404. (yes, it printed {x: 0, y: 0, z: 0 }.)